addons/payment_mollie_official/models/payment_acquirer.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+from mollie.api.client import Client
+from odoo import models, fields, api, _
+from odoo.http import request
+
+import base64
+import requests
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class PaymentAcquirer(models.Model):
+    _inherit = 'payment.acquirer'
+
+    _mollie_client = Client()
+
+    provider = fields.Selection(selection_add=[('mollie', 'Mollie')])
+    mollie_api_key_test = fields.Char(
+        'Mollie Test API key', size=40, required_if_provider='mollie',
+        groups='base.group_user')
+    mollie_api_key_prod = fields.Char('Mollie Live API key', size=40,
+                                      required_if_provider='mollie',
+                                      groups='base.group_user')
+    dashboard_url = fields.Char(string="Dashboard URL")
+    method_ids = fields.One2many('payment.acquirer.method',
+                                 'acquirer_id', 'Supported methods')
+
+    @api.model
+    def _get_main_mollie_provider(self):
+        return self.sudo().search([('provider', '=', 'mollie')], order="id",
+                                  limit=1) or False
+
+    @api.onchange('mollie_api_key_test')
+    def _onchange_mollie_api_key_test(self):
+        if self.mollie_api_key_test:
+            if not self.mollie_api_key_test[:5] == 'test_':
```

Comment:
remove these onchanges please, not needed

Reply:
The onchanges have been removed.